galette/templates/default/elements/modes.html.twig

```diff
@@ -1,11 +1,12 @@
 {% if login.isSuperAdmin() %}
-    <div class="ui small red inverted center aligned segment" title="{{ _T("You are actually logged-in as superadmin. Some functionnalities may not be available since this is *not* a regular member.") }}">
-        <i class="ui user shield icon"></i>
-        <strong>{{ _T("Superadmin") }}</strong>
-    </div>
+    <span class="ui red icon circular label">
+        <i class="tooltip shield icon" title="{{ _T("You are actually logged-in as superadmin. Some functionnalities may not be available since this is *not* a regular member.") }}"></i>
```

Comment:
I do not like title on `i` element only, I think it's more easy to be find on the whole element.

I've tried moving it to the parent element, and move also the `tooltip` class, but the display is not correct that way, I got standard browser tooltip... I don't know how to get that fixed :/

```suggestion
    <span class="ui red icon circular label tooltip" title="{{ _T("You are actually logged-in as superadmin. Some functionnalities may not be available since this is *not* a regular member.") }}">
        <i class="shield icon"></i>
```

Reply:
> I do not like title on `i` element only, I think it's more easy to be find on the whole element.

You're right !

> I've tried moving it to the parent element, and move also the `tooltip` class, but the display is not correct that way, I got standard browser tooltip... I don't know how to get that fixed :/

JS is required.